scene/animation/tween.cpp

```diff
@@ -778,14 +779,14 @@ bool Tween::resume_all() {
 }
 
 bool Tween::remove(Object *p_object, StringName p_key) {
-	_remove(p_object, p_key, false);
+	_remove(p_object, p_key);
```

Comment:
What's the point of having both `remove` and `_remove` if the only different is that the former already returns true (which is not particularly useful...)?

Seems to me like `_remove` could be merged into `remove` and removed :)
Ideally it should be `void Tween::remove` since the return value is meaningless, but to avoid breaking compatibility it's probably best to keep it as `bool Tween::remove`.

Reply:
You're right, I will merge them and keep `bool Tween::remove`.